tests/manage/z_cluster/nodes/test_disk_failures.py

```diff
@@ -365,6 +368,12 @@ def test_recovery_from_volume_deletion(self, nodes, pvc_factory, pod_factory):
             osd_pv.delete()
             osd_pv.ocp.wait_for_delete(resource_name=osd_pv_name)
 
+        # If we use LSO, we need to create and attach a new disk manually
```

Comment:
as discussed offline, this test is now adjusted to LSO VMDK.
Thinking we should have an issue to track the implementation of LSO PCI and LSO RDM

Reply:
Create the issue https://github.com/red-hat-storage/ocs-ci/issues/3583